include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -735,7 +735,7 @@ __parallel_radix_sort_iteration(_ExecutionPolicy&& __exec, ::std::size_t __segme
                                 sycl::event __dependency_event)
 {
     // Injecting ascending / descending status into custom name to prevent clashing kernel names
-    using _Ascending = std::conditional_t<__is_comp_asc, std::true_type, std::false_type>;
+    using _Ascending = std::bool_constant<__is_comp_asc>;
```

Comment:
@danhoeflinger, why don't we use `__radix_bits` parameter for kernel name? this is the only one that seems to be missing.

Reply:
@rarutyun  I believe in the current implementation `__radix_bits` is hardcoded to 4.  However this code should not make that assumption.  If `__radix_bits` that becomes variable, then that will cause name clashing.  It makes sense to add `__radix_bits`, especially because of the changes being made to radix sort currently.

Reply:
And yes I agree bool_constant is better.  :)